lib/rules/await-async-utils.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
```

Comment:
This rule implementation is almost the same as `await-async-query` but I didn't try to optimize this as we may migrate this to TS soon.